test/test-eth-trezor-keyring.js

```diff
@@ -128,16 +132,22 @@ describe('TrezorKeyring', function () {
     })
 
     it('should call TrezorConnect.getPublicKey if we dont have a public key', async function () {
+      const getPublicKeyStub = sinon.stub(TrezorConnect, 'getPublicKey').callsFake(
```

Comment:
At some point in the future it might be worth updating this keyring to inject the `TrezorConnect` instance, but use the direct import as a default parameter, so we can avoid stubbing globals like this in unit tests.  Stubbing globals can cause some pretty mysterious behaviour, especially in a test suite that has async tests that might continue doing things after the test ends.

Not something to fix in this PR though.